archeryutils/handicaps/handicap_scheme.py

```diff
@@ -266,7 +254,7 @@ def _s_bar(
 
         return max_score - sum(
             score_drop * np.exp(-(((arw_rad + (ring_diam / 2)) / sig_r) ** 2))
-            for ring_diam, score_drop in zip(ring_sizes, score_drops)
+            for ring_diam, score_drop in zip(ring_sizes, score_drops, strict=True)
```

Comment:
Doesn't hurt to have the strict check here in case of future fiddling with the implementation, although from how the current version works the two iterators will always be of length N and N + 1 - 1 = N.

Reply:
Done to satisfy Ruff B905: https://docs.astral.sh/ruff/rules/zip-without-explicit-strict/